velox/docs/develop/operators.rst

```diff
@@ -232,13 +232,54 @@ followed by the group ID column. The type of group ID column is BIGINT.
      - Description
    * - groupingSets
      - List of grouping key sets. Keys within each set must be unique, but keys can repeat across the sets.
+     - Grouping keys are specified with their output names.
    * - groupingKeyInfos
      - The names and order of the grouping key columns in the output.
    * - aggregationInputs
      - Input columns to duplicate.
    * - groupIdName
      - The name for the group-id column that identifies the grouping set. Zero-based integer corresponding to the position of the grouping set in the 'groupingSets' list.
 
+To illustrate why GroupingSets should use output column names lets examine the following SQL query:
+.. code-block:: sql
+    select COUNT(orderkey), count(distinct orderkey) from orders as t(orderkey);
```

Comment:
Would you generate the docs locally and verify formatting. I think you may need an empty line.